### PR TITLE
#125 Script Consistency and Ease of Use

### PR DESCRIPTION
**Pull request by C-Love**
Updates to help add functionality and support to the scripts. Additionally, add some useful functionality to the templates for a new script and closing project documentation. 

These templates will take the place of a function to 'get_this_script_started'. 

There will be a separate pull request to remove the start of the function from the FuncLib - but the file was in a branch when this part was done.

### CODE REVIEW

File: TYPE - NEW SCRIPT TEMPLATE.vbs [hunk 1]
```diff
@@ -52,10 +52,18 @@ changelog_display
...
+'  -- OR --
+MAXIS_footer_month = CM_plus_1_mo									'Directly assigns a footer month based on the current month
+MAXIS_footer_year = CM_plus_1_yr
 
 Dialog1 = "" 'blanking out dialog name
 'Add dialog here: Add the dialog just before calling the dialog below unless you need it in the dialog due to using COMBO Boxes or other looping reasons. Blank out the dialog name with Dialog1 = "" before adding dialog.
+'    Some Dialog Elements:  Initial Dialog Header: 			BeginDialog Dialog1, 0, 0, 191, 105, "CATEGORY - NAME Case Number Dialog"  				-- Use CATEGORY - NAME somewhere in the header
+'							Script Instructions Button:		PushButton 135, 5, 50, 15, "Instructions", script_instructions_btn						-- Have a button to open the instructions
+'							Script Purpose/Overview: 		Text 10, 70, 120, 30, "Here is a quick summary of the purpose of the script."			-- Give the worker a little guidance
+'							Include edit boxes for necessary detials like Case Number, Footer Month, Footer Year, and Worker Signature
```
Comment (MDR22) [inline TYPE - NEW SCRIPT TEMPLATE.vbs:66]: minor typo - details

**Review by MDR22 — APPROVED** [review]
Looks good! I like the changes.

**Review by IlseFerris — APPROVED** [review]
Besides Mark's comment, it all looks good. I approve thee!

**Review by MeganGeissler — APPROVED** [review]
Looks good to me. I couldn't think of anything else to add at this time!

File: TYPE - NEW SCRIPT TEMPLATE.vbs [hunk 2]
```diff
@@ -67,7 +75,11 @@ DO
         Call validate_footer_month_entry(MAXIS_footer_month, MAXIS_footer_year, err_msg, "*")   'IF NEEDED
         'The rest of the mandatory handling here
         IF trim(worker_signature) = "" THEN err_msg = err_msg & vbCr & "* Please sign your case note." 'IF NEEDED
-        IF err_msg <> "" THEN MsgBox "*** NOTICE!***" & vbNewLine & err_msg & vbNewLine
+		If ButtonPressed = msg_show_instructions_btn Then
```
Comment (MeganGeissler) [inline TYPE - NEW SCRIPT TEMPLATE.vbs:78]: Minor comment you use msg_show_instructions_btn  in the error handling and script_instructions_btn in your draft dialog btn var - might want to make them the same.

Comment (C-Love) [reply TYPE - NEW SCRIPT TEMPLATE.vbs]: Good catch!

File: TYPE - NEW SCRIPT TEMPLATE.vbs (hunk 1 above)
Comment (C-Love) [reply TYPE - NEW SCRIPT TEMPLATE.vbs]: Thanks!

**Pull request merged**